facebook/src/com/facebook/login/LoginManager.java

```diff
@@ -380,7 +380,8 @@ public boolean onActivityResult(int resultCode, Intent data) {
 
         if (!started) {
             FacebookException exception = new FacebookException(
-                    "Log in attempt failed: LoginActivity could not be started");
+                    "Log in attempt failed: FacebookActivity could not be started."
+                    + "Have you added FacebookActivity to your AndroidManifest?");
```

Comment:
Please add a space at the beginning, and update to:

    + " Please make sure you added FacebookActivity to the AndroidManifest."